content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,276 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a model registry to manage the lifecycle of the models we get from
+our experiments. Using DVC and DVC [Studio](/doc/studio) we will set up a model
+registry where we can discover, share, deploy and audit all our models and which
+will serve as the single source of truth for our model management.
+
+To make use of our model registry, we will need some trained models first. To
+speed things up, we will start from a git
+[repository](https://github.com/iterative/example-get-started-model-management)
+with a model training pipeline already set up and ready to use.
+
+To see how you can track experiments and set up training pipelines with DVC and
+DVCLive, have a look at our getting started guide for
+[experiment management](/doc/start/experiments).
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to save the
+model and add it to the model registry. Open the training script `src/train.py`
+in our example repository and you will see the following code under the
+`with Live(...)` context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+The training script runs as a part of a dvc pipeline
+[we prepared](/doc/start/experiments/experiment-pipelines) in the Experiment
+Management guide, so now we just call `dvc exp run` to run the experiment.
```

Comment:
This reads to me like `dvc exp run` is a requirement, or at least it's unclear what to do if I don't already know about `dvc exp run`.

Reply:
Good point. I had the pipeline run from a notebook at first, but I didn't like the fact that for something already production-oriented we are using notebooks in the example. And this way we also can more easily continue with the steps in the getting-started-experiments repo.

I guess I can reword that though to mention how you would do it in general (just run the code in any way) and ho we do it in the example where we already have the pipeline.